src/main/resources/view/MainWindow.fxml

```diff
@@ -13,7 +13,7 @@
 <?import javafx.scene.layout.VBox?>
 <?import javafx.stage.Stage?>
 
-<fx:root minHeight="700.0" minWidth="1000.0" onCloseRequest="#handleExit" title="SOCareers" type="javafx.stage.Stage" xmlns="http://javafx.com/javafx/20.0.1" xmlns:fx="http://javafx.com/fxml/1">
+<fx:root maximized="true" minHeight="700.0" minWidth="1300.0" onCloseRequest="#handleExit" title="SOCareers" type="javafx.stage.Stage" xmlns="http://javafx.com/javafx/20.0.1" xmlns:fx="http://javafx.com/fxml/1">
```

Comment:
Unfortunately due to the feature freeze, I don't think we're allowed to change the UI :(